http/api.go

```diff
@@ -157,7 +157,10 @@ func HandleBattleSimulate(w http.ResponseWriter, r *http.Request) {
 
 	bytes, err := json.Marshal(results)
 	if err != nil {
-		log.Fatalf("Failed to marshal into JSON: %s", err)
+		log.Printf("Failed to marshal into JSON: %s", err)
```

Comment:
is there a reason this isn't a Panicf like the similar JSON error in agent.go?

Reply:
The panic in agent.go is to get it to leave the function and fail. The print here is to let the server respond to the request.

Reply:
makes sense!